src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/global.jelly

```diff
@@ -61,16 +61,16 @@
 
       <f:validateButton title="Test SSC connection" progress="Testing..." method="testConnection" with="url,token,useProxy,proxyUrl,proxyUsername,proxyPassword" />
       <f:entry title="Maximum issues per page" help="/plugin/fortify/help-breakdownPageSize.html">
-        <f:textbox field="breakdownPageSize" id="breakdownPageSize" />
+        <f:number field="breakdownPageSize" id="breakdownPageSize" />
       </f:entry>
       <f:entry title="Connection timeout" help="/plugin/fortify/help-connectionTimeout.html">
```

Comment:
What is the unit for this?

Reply:
You can check out setting helper (question mark icon) where the information about the unit of the field is specified.